torch-sys/build.rs

```diff
@@ -7,9 +7,19 @@
 // like 9.0, 90, or cu90 to specify the version of CUDA to use for libtorch.
 
 use anyhow::{Context, Result};
+
+use std::ffi::OsStr;
+use std::os::unix::ffi::OsStrExt;
 use std::path::{Path, PathBuf};
 use std::{env, fs, io};
 
+#[cfg(feature = "mobile")]
+use base64::engine::general_purpose;
+#[cfg(feature = "mobile")]
+use base64::Engine;
+#[cfg(feature = "mobile")]
+use std::env::var;
+
```

Comment:
If possible, I'd recommend moving feature-gated imports into the functions they are used in, so you don't have to separately feature-gate each import and ensure that they stay in sync with where they are used. You can just have use-statements inside functions.